AISKU/Tests/ApplicationInsightsDeprecatedTests.ts

```diff
@@ -15,19 +19,18 @@ export class ApplicationInsightsDeprecatedTests extends TestClass {
     private clearSpy: SinonSpy;
     private _name = "ApplicationInsightsDeprecatedTests: ";
     
+    constructor() {
+        super("ApplicationInsightsDeprecatedTests");
+    }
     public testInitialize() {
         try {
-
             this.useFakeServer = false;
-            (sinon.fakeServer as any).restore();
-            this.useFakeTimers = false;
-            this.clock.restore();
-
             this._snippet = {
                 config: {
                     instrumentationKey: ApplicationInsightsDeprecatedTests._instrumentationKey,
                     disableAjaxTracking: false,
-                    disableFetchTracking: false
+                    disableFetchTracking: false,
+                    maxBatchInterval: 2500
```

Comment:
This is just to speed up the test execution, so it doesn't have to wait for the default 15 seconds before the Sender sends the request